### PR TITLE
add dask-tensorflow

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @rsignell-usgs (let me know if you need more recipes for the workshop).

### CODE REVIEW

File: recipes/dask-tensorflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+about:
+  home: The package home page
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Interactions between Dask and Tensorflow
+
+extra:
+  recipe-maintainers:
+    - mrocklin
```
Comment (ocefpaf) [inline recipes/dask-tensorflow/meta.yaml]: @mrocklin is it OK to add you here?

**Comment by ocefpaf** [comment]
@jjhelmus I had this issue when build this locally with the docker image:

```
import: 'dask_tensorflow'
Traceback (most recent call last):
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: /lib64/libc.so.6: version `GLIBC_2.16' not found (required by /opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/test_tmp/run_test.py", line 2, in <module>
    import dask_tensorflow
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/dask_tensorflow/__init__.py", line 1, in <module>
    from .core import _start_tensorflow, start_tensorflow
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/dask_tensorflow/core.py", line 8, in <module>
    import tensorflow as tf
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "/opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: /lib64/libc.so.6: version `GLIBC_2.16' not found (required by /opt/conda/conda-bld/dask-tensorflow_1528804470108/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
```

Am I doing something wrong or it `tensorflow` broken?

**Comment by mrocklin** [comment]
I'm somewhat surprised that someone is requesting a recipe for this.  It's a fairly small and, as far as I know, unused library.

That being said, yes, I'm happy to be listed as maintainer.  @TomAugspurger would be another good choice here.

**Comment by rsignell-usgs** [comment]
@mrocklin we are interested exploring this for potential use for several workshops on machine learning that USGS is leading this summer, and would like it on conda-forge so we can build our conda environment using only conda-forge as an additional channel.

**Comment by TomAugspurger** [comment]
Feel free to add me as well.

**Comment by ocefpaf** [comment]
Thanks @TomAugspurger! Just added you.

@jjhelmus nevermind about https://github.com/conda-forge/staged-recipes/pull/6070#issuecomment-396566716 my memory failed me for a while but now I remember that we are re-packaging the google's wheel.

We can ignore the linter warning here b/c the skip there is only for the docker image and not really an `arch` limitation. Users with modern Linux should be able to install this (actually `tensorflow`) without a problem.

**Comment by jjhelmus** [comment]
> Users with modern Linux should be able to install this (actually tensorflow) without a problem.

Note that "modern" Linux is Ubuntu 16.04 and later.  The `tensorflow` conda packages in `defaults` will work on CentOS 6 and above.

**Comment by ocefpaf** [comment]
> The tensorflow conda packages in defaults will work on CentOS 6 and above.

Any chance on getting those in conda-forge too :wink:

**Pull request merged**